dev-docs/source/ReleaseChecklist.rst

```diff
@@ -15,260 +15,327 @@ Roles
 
+*  Create issues for people to neaten up the release notes and add images etc.
 
-Wednesday, Release- 2 weeks & 3 days
-####################################
+Tuesday, 4 days
+---------------
 
-**Task Priorities**: Blocker bug fixes, Testing, Release Notes,  Maintenance Tasks, Next release development.
+*  Review the complete set of release notes to make sure there are no glaring mistakes.
 
-Beta Test Open
---------------
+.. _release-manager-checklist:
 
-*  Before sending an email to users, ensure that the Usage data .zip
-   file containing usage data is up-to-date. This is done by downloading
-   the current .zip from sourceforge, adding any missing files, and
-   resending it.
-*  Send an email to beta test users explaining where to download the
-   installers and how to report issues (cc the Local PMs so they can organise a similar message at their facilities).
-*  Developers to arrange to meet with their beta testers.
-*  Create issues for people to neaten up the release notes and add images etc.
+Release Manager Checklist
+#########################
 
-Wednesday, Release- 1 week & 3 days
-###################################
+**Role**: Person in charge of the go/no go decision of the release. The main task 
+is to reiterate the timeline and be the collection point for information between 
+all of the Local Project Managers.
 
-**Task Priorities**: Blocker bug fixes, Testing, Release Notes,  Maintenance Tasks, Next release development.
+Friday, 4 weeks & 1 day
+-----------------------
 
-Beta Test Reminder
-------------------
+*  Post on the *\#general* slack channel reminding developers of the impending 
+   release and stating that they have only 5 days left before the code freeze.
+*  Send an email to beta test users explaining the dates for the testing, and 
+   stating they will have more detail on the start of the first day (cc the Local 
+   Project Manager(s) so they can organise a similar message at their facilities).
 
-*  Send an email to beta test users thanking them for there feedback so far and reminding them to feedback as soon as possible
-   and not to send in a list of issues at the end of testing (cc the Local PMs so they can organise a similar message at their facilities).
+Friday, 3 weeks & 1 day
+-----------------------
 
+*  Post on the *\#general* slack channel asking everyone to ensure they have moved 
+   any incomplete issues to the next milestone, stating the code freeze is in place, 
+   and warning developers that non-blocker issues will be moved from the milestone 
+   on Monday morning.
+*  Attempt to drive the pull requests for this milestone down to 0, in collaboration
+   with the Local Project Managers.
 
-Tuesday, Release- 4 days
-########################
+Monday, 3 weeks
+---------------
+
+*  Ensure that PR testing has been completed for PRs from before the code freeze.
+
+**Clearing the project board**
+
+Go through the Zenhub project board for the release milestone (not the sprint 
```

Comment:
Could we remove the mention of Zenhub and just say "Go through the issues for the release milestone"?